odl_video/settings.py

```diff
@@ -182,3 +182,27 @@
 VIDEO_S3_BUCKET = os.environ.get('VIDEO_S3_BUCKET', 'odl-video-service')
 VIDEO_S3_TRANSCODE_BUCKET = os.environ.get('VIDEO_S3_TRANSCODE_BUCKET', '{}_transcoded'.format(VIDEO_S3_BUCKET))
 VIDEO_S3_THUMBNAIL_BUCKET = os.environ.get('VIDEO_S3_THUMBNAIL_BUCKET', '{}_thumbnails'.format(VIDEO_S3_BUCKET))
+
+# AWS S3 upload settings
+# the defaults values come from the default configuration in boto3.s3.transfer.TransferConfig
+# apart from the first 2
```

Comment:
I made some tests and the default value of 8MB seems to not be optimal for the upload.